go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -665,6 +665,9 @@ func TestApplyVSchema(t *testing.T) {
 							Sharded: false,
 						},
 					},
+					MirrorRules: &vschemapb.MirrorRules{
+						Rules: []*vschemapb.MirrorRule{},
+					},
```

Comment:
Is it possible for people to provide MirrorRules through ApplyVSchema?

Reply:
hm, it should not be possible any more than it is possible to change routing rules through `ApplyVSchema`.